examples/entity-viewer-v2/entity-viewer-client/src/main/ml-modules/root/explore-data/ui-config/config.json

```diff
@@ -633,8 +633,29 @@
         }
     },
 
-    "occupations": {}
+    "occupations": {},
 
+    "membership": {
+      "component": "Relationships",
+      "config": {
+        "arrayPath": "person.memberships.membership",
+        "iconSize": 35,
```

Comment:
Let's add a date configuration here:
```
"dateFormat": "yyyy-MM-dd",
```
Then we can use it with our DateTime component.